hotsos/defs/tests/scenarios/storage/ceph/ceph-mon/empty_clog.yaml

```diff
@@ -0,0 +1,46 @@
+target-name: empty_clog.yaml
```

Comment:
target-name isnt needed if the filename is identical. doesn't hurt tho of course.